src/odin/enhancedtrippath.cc

```diff
@@ -1761,5 +1761,16 @@ std::string EnhancedTripLeg_Admin::ToString() const {
   return str;
 }
 
+const google::protobuf::RepeatedPtrField<valhalla::TripLeg_Node_Incident> empty;
+
+const google::protobuf::RepeatedPtrField<valhalla::TripLeg_Node_Incident>&
+EnhancedTripLeg_Node::incidents() const {
+  if (mutable_node_) {
+    return mutable_node_->incidents();
+  } else {
+    return empty;
```

Comment:
im not sure if it works for repeated fields but protobuf has t his thing called default_instance. you might be able to do something like: `return google::protobuf::RepeatedPtrField<valhalla::TripLeg_Node_Incident>::default_instance;`

Reply:
Doesn't seem to exist for repeatedPtrField.